### DISCUSSION TITLE
Angular pages (other than "/") 404ing in prod

### DISCUSSION THREAD

**Discussion by stef-timmermans** [community/community#58340] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi all,

I'm a senior CS student and I'm pretty new to using GitHub Pages to host websites. I have a pretty standard front-end Angular project here:

https://github.com/stef-timmermans/stef-timmermans.github.io

Routing info can be found in `src/app/app-routing.modules.ts`

If you go into prod (the deployment link on the repo), you'll find that the links to other pages do not work. However, in development, they do work. I made sure to run `ng build --configuration production` before pulling changes into main. I also have a ** handler for 404 pages, but that doesn't work either. Is this due to how GitHub Pages handles Angular routing? Any help with this would be appreciated.

**Comment by ng-druid** [comment, marked as answer]
It appears the custom domain has fixed your issue. For future reference --base-href option for ng build is required when the site root is under a path.

<img width="1440" alt="Screen Shot 2023-06-21 at 4 13 13 AM" src="https://github.com/community/community/assets/73197190/ab9daac6-f96f-440f-b223-41c529d3260c">

Working Example:

* https://rollthecloudinc.github.io/plugin-amcharts5-build/
